akka-persistence/src/main/scala/akka/persistence/snapshot/SnapshotStore.scala

```diff
@@ -44,10 +44,18 @@ trait SnapshotStore extends Actor {
       delete(metadata)
```

Comment:
is this still ok? should the `evt` be sent to the sender when the future is completed?